setup.py

```diff
@@ -89,7 +89,12 @@ def build_parser():
         action="store_true",
         help="Don't update git submodules",
     )
-
+    parser.add_argument(
+        "--config",
```

Comment:
```suggestion
        "--build-type",
```
`config` is not a very descriptive name